manifests/repo.pp

```diff
@@ -143,11 +143,11 @@
           require => Exec["import_gpg_${normalized_name}"],
         }
 
-        exec { "yum_make_cache_${repo_name}":
-          command => "yum -q makecache -y --disablerepo='*' --enablerepo='${normalized_name}'",
-          path    => '/usr/bin',
-          require => File[$normalized_name],
```

Comment:
Yes, but not needed and breaks puppet idempotency since it runs every
single time.  Yum normally takes care of the cache itself so not sure why
this would be needed.

Dan

On Wed, Feb 24, 2016 at 3:16 PM, Peter Souter <notifications@github.com>
wrote:

> In manifests/repo.pp
> <https://github.com/computology/computology-packagecloud/pull/19#discussion_r54007610>
> :
>
> > @@ -143,11 +143,11 @@
> >            require => Exec["import_gpg_${normalized_name}"],
> >          }
> >
> > -        exec { "yum_make_cache_${repo_name}":
> > -          command => "yum -q makecache -y --disablerepo='*' --enablerepo='${normalized_name}'",
> > -          path    => '/usr/bin',
> > -          require => File[$normalized_name],
>
> Isn't the idea to ensure the latest cache has been build against the
> remote yumrepo?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/computology/computology-packagecloud/pull/19/files#r54007610>
> .
>



-- 

*Daniel Wittenberg*  *|*  *Sr. Technical Architect*  *|*  *AHEAD*

Work: 312.508.4996  |  Mobile: 319.329.9776  |  Fax: 312.924.5076

dan.wittenberg@ThinkAHEAD.com  |  ThinkAHEAD.com
<http://info.thinkahead.com/e1t/c/*W6xgnfb3P660yN6FC2m6RB8ng0/*VZbrYd83jBHmW8yYcRc7tsC9Q0/5/f18dQhb0S5fw8XJ8hgW7-KP3h2qwv1yN32Gdlh5YWcwMf59TdXD6prW39Dr-N8pCDJGVYwxWD50wxKSW1wSwgX6PZxrZW1nrCGw96zRPSW6bT6L38RRjRDW5Ckvpl6vYDb4W7v70tS4RYYVxW3s1XjF51G0BFN4-0Bs5YSX-5W7m_B1R5c8d-SW377W4T2y-PYYW1chfKP7cMWqbW7yHnNh5v7qjJW8f27Mv160vTSW3JX9bz6QCkN4W3yrKmV25vJtRN6RRKCDDSjH1W1x2Rdy3scNzKW7cLGH22_P0-DW3TgRkj2BH5L0W3MfK-q6QtBYkW4VsJNn5tRSXLW2ptwYT7-18V4W3Jyb2b5c1xDZM9XCtYhzVNsW75f7r78csGDVVD2JF57dZRWvW5BbJtH3fDF-GW30D2Nn2tf-5XW4Wrj0H31GHfxW11PXZW1DylWwN91kcK-5LD0kW85GKv28HFk6-VGJMbw4x6kZpW4L1MlS4HzlXXf3hvFt-11>

Subscribe to the AHEAD Blog
<http://info.thinkahead.com/e1t/c/*W6xgnfb3P660yN6FC2m6RB8ng0/*W4H9N4R5sYPcLW3_0Jl86g_BN10/5/f18dQhb0S5fk8YXMz-W7-KP3h2qwv1yN32Gdlh5YWcwMf5p52XD6prW39DrXV26p8nbW1mbT-97ZcDx5W50F6ch96LsblW61SSZm7dDxTNW7mWsnX7dzcsSW51vDDW56vHg5W69NG1w6Pr3nqW1Tcntw1njMtBW6H5Hs77_P6JSW52Ylfx3_8mplW9fgwMT4fPvb6W7pRv1n6h0Y3ZW9cjGmB1jFrM1W4BFGrL1jwGCCN2N3s5RskrlmW56jSgc7JvgQsW78kBlP3q47m-V67vPz2P5LXMW4C58BM7vnhmHN1jYmWVnQZTcW1xV_1P9cCHJhW30b2vC4hn41nW4B8qzr7psv71N4PzVjXLrJxBN7Ll39CmMR98W7lbVPS5KXkt3N8csGDVD2JF5W7dZRWv5BbJtHW3fDF-z30D2NnN4R7Y0q9VByCW8TbH5T2Csyq-W3smpnF9bW0MlW8tzLQV7FhBvSN4QYsZjT-Y0ZW6NGkHS2pgj48V1-H838nFWNQ102>